packages/contracts/scripts/generate-typechain.ts

```diff
@@ -0,0 +1,27 @@
+import {exec} from 'child_process';
```

Comment:
Didn't we already have a very similar script for this job?

Reply:
yes but was removed from contracts-versions
and i refactored it here